hal/src/main/native/athena/FRCDriverStation.cpp

```diff
@@ -229,6 +229,18 @@ int32_t HAL_SendError(HAL_Bool isError, int32_t errorCode, HAL_Bool isLVCode,
   return retval;
 }
 
+int32_t HAL_SendConsoleLine(const char* line) {
+  wpi::StringRef lineRef{line};
+  if (lineRef.size() <= 65536) {
+    // Send directly
+    return FRC_NetworkCommunication_sendConsoleLine(line);
+  } else {
+    // Need to truncate
+    std::string newLine{line, 65536};
```

Comment:
It probably needs to be a bit shorter than this for header bytes?

Reply:
Nope. The header bytes are not included in the length parameter. So this is just to limit the lenth to ushort max. Same thing was done for errors.

Reply:
Wouldn't ushort max be 65535?